mri/operators/fourier/non_cartesian.py

```diff
@@ -498,7 +498,7 @@ def _adj_op(self, coeff):
         return stacked_images * np.sqrt(self.num_slices / self.acq_num_slices)
 
     def adj_op(self, coeff):
-        """ This method calculates inverse masked non-uniform Fourier
+        """ This method computes inverse masked non-uniform Fourier
```

Comment:
(D401) First line should be in imperative mood.
(D210) No whitespaces allowed surrounding docstring text
```suggestion
        """Compute  the inverse masked non-uniform Fourier
```